dask_expr/io/io.py

```diff
@@ -121,6 +126,37 @@ def _combine_similar(self, root: Expr):
         return
 
 
+class FusedIO(BlockwiseIO):
+    _parameters = ["expr"]
```

Comment:
I suppose we will still generate the graph for `self.operand("expr")` when `FusedIO.__dask_graph__` is called.